flask_jsonlocale.py

```diff
@@ -6,26 +6,26 @@
 
 
 class Locales(object):
-    def __init__(self, app):
+    def __init__(self, app, app_dir):
         self.app = app
-        self.init_app(app)
+        self.init_app(app, app_dir)
         self._messages = {}
-    
-    def init_app(self, app):
-        app.config.setdefault('MESSAGES_DIR', 'messages')
+
+    def init_app(self, app, app_dir):
+        app.config.setdefault('MESSAGES_DIR', app_dir+'\messages')
```

Comment:
Beware – this is not cross-platform. Unix-based systems don't use backwards slashes.